vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/UserExportIT.java

```diff
@@ -0,0 +1,148 @@
+        admin.move(test.getPath(), TEST_USER_PATH);
+        admin.save();
+        assertNotNull(getTestUser());
+        log.info("Test user created at path {}", test.getPath());
+        byte[] serialised = export(admin, TEST_USER_PATH);
+        File tmpFile = File.createTempFile("test", ".zip");
+        try (FileOutputStream fso = new FileOutputStream(tmpFile)) {
+            fso.write(serialised);
+        }
+        log.info("Test user exported at path {}", tmpFile.getAbsolutePath());
+        clean(TEST_USER_PATH);
+        assertNull(getTestUser());
+        log.info("Test user removed from the repository");
+        importPackage(admin, new ByteArrayInputStream(serialised));
+        assertNotNull(getTestUser());
+        log.info("Test user imported");
+    }
+
+    private User getTestUser() throws RepositoryException {
+        return (User) ((JackrabbitSession) admin).getUserManager().getAuthorizable(TEST_USER_ID);
+    }
+
+    private void importPackage(Session session, InputStream is) throws IOException, ConfigurationException, RepositoryException, PackageException {
+        ImportOptions opts = new ImportOptions();
+        opts.setAccessControlHandling(AccessControlHandling.OVERWRITE);
+        opts.setCugHandling(AccessControlHandling.OVERWRITE);
+        opts.setImportMode(ImportMode.REPLACE);
+        opts.setPatchKeepInRepo(false);
+        opts.setAutoSaveThreshold(-1);
+        opts.setStrict(true);
+        Importer importer = new Importer(opts);
+        try (Archive archive = new ZipStreamArchive(is)) {
+            archive.open(false);
+            importer.run(archive, session, "/");
+            if (importer.hasErrors() && opts.isStrict(true)) {
+                throw new RuntimeException("Failed to import");
+            }
+        }
+    }
+
+
+    private byte[] export(Session session, String authorizablePath) throws IOException, RepositoryException, ConfigurationException {
+
+        PathFilterSet nodeFilters = new PathFilterSet(authorizablePath);
+        PathFilterSet propertyFilters = new PathFilterSet(authorizablePath);
+
+        DefaultWorkspaceFilter filter = new DefaultWorkspaceFilter();
+        filter.add(nodeFilters, propertyFilters);
+
+        DefaultMetaInf inf = new DefaultMetaInf();
+        inf.setFilter(filter);
+
+        Properties props = new Properties();
+        props.setProperty(VaultPackage.NAME_GROUP, "jackrabbit/sync");
+        props.setProperty(VaultPackage.NAME_NAME, randomUUID().toString());
+        props.setProperty(VaultPackage.NAME_VERSION, "0.0.1");
+        props.setProperty(PackageProperties.NAME_USE_BINARY_REFERENCES, String.valueOf(true));
+        inf.setProperties(props);
+
+        ExportOptions opts = new ExportOptions();
+        opts.setMetaInf(inf);
+
+        opts.setRootPath(authorizablePath);
```

Comment:
it works once you comment this and the following line. If you really need those options they should be in platform format (not in repo format I guess).

Reply:
I proposed a fix in together with this test in #336